osbrain/proxy.py

```diff
@@ -246,7 +246,8 @@ def _is_safe_method(self, methodname):
                 methodname not in ('run', 'get_attr', 'kill', 'safe_call',
                                    'concurrent', 'is_running'))
 
-    def _remote_call(self, methodname, args, kwargs, flags, objectId):
+    def _remote_call(self, methodname, args, kwargs, flags,  # noqa: N803
+                     objectId):
```

Comment:
Cannot we put the `  # noqa` comment after the `:`?

Reply:
Nope. I tried that already and it doesn't work :disappointed: